.circleci/src/jobs/deliver_ios.yml

```diff
@@ -0,0 +1,34 @@
+# Deliver the app to the Apple App Store or Testflight.
+parameters:
+  buildConfig:
+    type: enum
+    enum: [bayern, nuernberg]
+  production_delivery:
+    description: Whether to deliver the build to production.
+    type: boolean
+macos:
+  xcode: 15.0.1
```

Comment:
is there a reason why we use different xcode versions? First `14.3.1` and now `15.0.1`

Reply:
For delivery 15.0.1 works fine, for building i needed to downgrade, due to a bug in `activesupport`.